src/environments/environment.ci.ts.template

```diff
@@ -1,61 +1,62 @@
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with this program.  If not, see https://www.gnu.org/licenses/.
- */
+* AMRIT – Accessible Medical Records via Integrated Technology
+* Integrated EHR (Electronic Health Records) Solution
+*
+* Copyright (C) "Piramal Swasthya Management and Research Institute"
+*
+* This file is part of AMRIT.
+*
+* This program is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 3 of the License, or
+* (at your option) any later version.
+*
+* This program is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with this program.  If not, see https://www.gnu.org/licenses/.
+*/
+
 
 // The file contents for the current environment will overwrite these during build.
 // The build system defaults to the dev environment which uses `environment.ts`, but if you do
 // `ng build --env=prod` then `environment.prod.ts` will be used instead.
 // The list of which env maps to which file can be found in `.angular-cli.json`.
 
-const SERVER_IP = '<%= SERVER_IP %>';
+const SERVER_IP =  '<%= SERVER_IP %>';
 const SWYMED_IP = '<%= SWYMED_IP %>';
 const ADMIN_API = '<%= ADMIN_API_BASE %>';
 const COMMON_API_OPEN = '<%= COMMON_API_BASE %>';
 const COMMON_API = '<%= COMMON_API_BASE %>';
-const MMU_API = '<%= MMU_API_BASE %>';
+const MMU_API =  '<%= MMU_API_BASE %>';
 const TM_API = '<%= TM_API_BASE %>';
 const COMMON_API_OPEN_SYNC = '<%= COMMON_API_OPEN_SYNC %>';
-const SCHEDULER_API = '<%= SCHEDULER_API_BASE %>';
-const SCHEDULER_UI = '<%= SCHEDULER_UI_BASE %>';
-const INVENTORY_UI = '<%= INVENTORY_UI_BASE %>';
-const IDENTITY_API = '<%= IDENTITY_API_BASE %>';
+const SCHEDULER_API =  '<%= SCHEDULER_API_BASE %>';
+const SCHEDULER_UI =  '<%= SCHEDULER_UI_BASE %>';
+const INVENTORY_UI =  '<%= INVENTORY_UI_BASE %>';
+const IDENTITY_API =  '<%= IDENTITY_API_BASE %>';
 
-const mmuUICasesheet = '<%= MMU_UI_BASE %>';
-const IOT_API = 'http://localhost:8085/ezdx-hub-connect-srv';
+const mmuUICasesheet ='<%= MMU_UI_BASE %>';
+const IOT_API = '<%= IOT_API_BASE %>';
 
 export const environment = {
   production: true,
-  isMMUOfflineSync: false,
+  isMMUOfflineSync: true,
 
   app: `MMU`,
   RBSTest: `RBS Test`,
   visualAcuityTest: `Visual Acuity Test`,
   haemoglobinTest: `Haemoglobin Test`,
   parentAPI: `${MMU_API}`,
 
-  INVENTORY_URL: `${INVENTORY_UI}/#/redirin?`,
+  INVENTORY_URL: `${INVENTORY_UI}inventory-ui-next/#/redirin?`,
```

Comment:
No, we don't want inventory-ui-next here.
Instead {INVENTORY_UI} set in env will have the base URL detail.
Previous implementation (removed line) is correct.

Reply:
@drtechie HI Mithun, updated as per comments, Please check